src/entity/prelude.rs

```diff
@@ -36,6 +36,39 @@ pub type DateTimeUtc = chrono::DateTime<chrono::Utc>;
 #[cfg(feature = "with-chrono")]
 pub type DateTimeLocal = chrono::DateTime<chrono::Local>;
 
+#[cfg(feature = "with-chrono")]
```

Comment:
is there an abstract way which would do this faster rather than importing the `#[cfg()]` macro always.